csrc/transformer/dropout_kernels.cu

```diff
@@ -690,14 +692,15 @@ void launch_dropout(T* out,
                     float ratio,
                     cudaStream_t stream)
 {
-    dim3 grid_dim(batch);     // DS_GET_BLOCKS(total_count/4);
-    dim3 block_dim(dim / 4);  // DS_CUDA_NUM_THREADS;
+    int total_count = batch * dim / 4;

```

Comment:
Can you use a constant variable explaining what 4 is?

Reply:
sure, 4 is the loop-unrolling degree.